app/app/controllers/cbv/synchronizations_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Cbv::SynchronizationsController < Cbv::BaseController
+  def show
+    account_id = params[:user][:account_id]
+
+    @pinwheel_account = PinwheelAccount.find_by_pinwheel_account_id(account_id)
```

Comment:
We need make sure the current user has permission to view this PinwheelAccount.

```suggestion
    @pinwheel_account = @cbv_flow.pinwheel_accounts.find_by(pinwheel_account_id: account_id)
```

^ should do the trick